CMSIS/NN/Include/arm_nnsupportfunctions.h

```diff
@@ -422,8 +433,65 @@ __STATIC_FORCEINLINE q31_t arm_nn_divide_by_power_of_two(const q31_t dividend, c
  */
 __STATIC_FORCEINLINE q31_t arm_nn_requantize(const q31_t val, const q31_t multiplier, const q31_t shift)
 {
-  return arm_nn_divide_by_power_of_two(arm_nn_sat_doubling_high_mult(val * (1 << LEFT_SHIFT(shift)), multiplier),
-                                       RIGHT_SHIFT(shift));
+  return DIV_POW2(MUL_SAT(val * (1 << LEFT_SHIFT(shift)), multiplier), RIGHT_SHIFT(shift));
```

Comment:
just here could you put the old names back? It is used in almost all functions and is nice with having a descriptive name and one level less to browse through.